python/feathub/metric_stores/metric_store.py

```diff
@@ -111,6 +116,13 @@ def create_metric_materialization_descriptors(
                     allow_overwrite=True,
                 )
             )
+        if feature_descriptor.keep_source_metrics:
+            descriptors.extend(
+                self.create_metric_materialization_descriptors(
+                    feature_descriptor=feature_descriptor.get_resolved_source(),
+                    data_sink=BlackHoleSink(),
```

Comment:
It seems that only the sink name is used to set the "table_name" label of the Prometheus metric. Why can't we use the feature view name instead?